webapp/tests/Unit/Controller/API/GeneralInfoControllerTest.php

```diff
@@ -160,7 +160,7 @@ public function testCountryFlagNotFound(string $countryCode, string $size)
             /** @var BinaryFileResponse $response */
             $response = $this->client->getResponse();
             static::assertEquals(404, $response->getStatusCode());
-            static::assertEquals(sprintf('country flag for %s of size %s not found', strtoupper($countryCode), $size), json_decode($response->getContent(), true)['message']);
+            static::assertEquals(sprintf('Country flag for %s of size %s not found.', strtoupper($countryCode), $size), json_decode($response->getContent(), true)['message']);
```

Comment:
Should you not also update the API which sets this value?

Reply:
I didn't want to include this in this PR.